setup.js

```diff
@@ -19,7 +19,7 @@ function resolve_local_file(file_path) {
 }
 
 function setup_virtualenv(on_success, on_error) {
-  exec('python -m pip install virtualenv && python -m virtualenv ' + resolve_local_file('.venv'),
+  exec('rm -r' + resolve_local_file('.venv') + ' || true && python -m pip install virtualenv && python -m virtualenv ' + resolve_local_file('.venv'),
```

Comment:
nit: use `rm -rf`, `-f` ignores non-existing files

Reply:
It also ignores file permissions, which makes it much more dangerous if `resolve_local_file` does not resolve correctly. I'd rather leave the command as is, since it does the job while also not removing what it's not supposed to.